webpack.config.js

```diff
@@ -152,7 +162,7 @@ const development = merge([
       host: '0.0.0.0',
       historyApiFallback: true,
       proxy: {
-        '/nuxeo': 'http://localhost:8080/',
+        '/nuxeo': `http://${process.env.NUXEO_HOST || 'localhost'}/`,
```

Comment:
Missing port here. If we assume `NUXEO_HOST` also includes the port, we could do something like:
```suggestion
        '/nuxeo': `http://${process.env.NUXEO_HOST || 'localhost:8080'}/`,
```